db/migrate/20161122035953_question_order_type.rb

```diff
@@ -0,0 +1,5 @@
+class QuestionOrderType < ActiveRecord::Migration
+  def change
+    add_column :question_orders, :type, :string, null: false
```

Comment:
What is `type` in `question_order` used for? I don't see it being referenced in this PR